Autopilot/PathManager/Inc/CommWithAttitudeManager.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef GET_FROM_PATH_MANAGER_HPP
+#define GET_FROM_PATH_MANAGER_HPP
+
+//Commands to send to attitude manager.
+struct CommandsForAM{
+	float roll,pitch,yaw;	// commanded orientation (radians)
+	float airspeed;			// commanded airspeed (m/s)
+};
+
+//Data to receive from attitude manager.
+struct AttitudeData{
+	float roll,pitch,yaw;	// current orientation (radians)
+	float airspeed;			// current airspeed (m/s)
+};
+
+/**
+ * Sends commands to the attitude manager.
+ * @param[out]   Commands   Pointer to struct with commands.
+ */
+void SendCommandsForAM(CommandsForAM *Commands);
+
+/**
+ * Retrieves attitude/airspeed data from attitude manager.
+ * @return Attitude/airspeed data.
+ */
+AttitudeData GetAttitudeData();
+
+#endif
```

Comment:
![image](https://user-images.githubusercontent.com/32182266/99565316-7a171400-2999-11eb-8e0d-db5314a268d8.png)
you should always add a new line at the end of your files. In posix, a line is defined as a string of characters ending with the new line character, so some compilers will throw warnings when you don't. 

Not anything very important, but may as well get in the habit of it (I'd suggest configuring your text editor to automatically add one on save or something.